### PR TITLE
Addition of Leaky Rectifier ReLU

### PR DESCRIPTION
**Pull request by isacarnekvist**

### CODE REVIEW

File: blocks/bricks/simple.py [hunk 1]
```diff
@@ -284,6 +284,28 @@ def apply(self, input_):
         return tensor.switch(input_ > 0, input_, 0)
 
 
+class LeakyRectifier(Activation):
+    r"""A linear transformation with optional bias.
```
Comment (dmitriy-serdyuk) [inline blocks/bricks/simple.py]: Seems like copy-paste from somewhere else. It's totally not true.

I think, documentation needs some citations.

**Comment by dmitriy-serdyuk** [comment]
Thank you! But I think it might be better to place this to [blocks-extras](https://github.com/mila-udem/blocks-extras/) since it's not so much widely used as ReLU.

I made some comments according to documentation. Also, this code needs some tests.

**Comment by dwf** [comment]
It's becoming quite widely used actually. I'd agree for parametric ReLU,
but I see ReLUs with a fixed leak more and more often.

**Comment by isacarnekvist** [comment]
I will add some tests for activation and gradients during the day as well. @dwf When you say parametric ReLU, do you mean the above simple LeakyReLU or the one where you can include a, and b from 

`f(x) = max(ax, bx)`

in the gradient so that you can learn these during training as well?

File: blocks/bricks/simple.py [hunk 2]
```diff
@@ -284,6 +284,29 @@ def apply(self, input_):
         return tensor.switch(input_ > 0, input_, 0)
 
 
+class LeakyRectifier(Activation):
+    r"""Like Rectifier, but instead does:
```
Comment (dmitriy-serdyuk) [inline blocks/bricks/simple.py]: Please take a look at [numpy docstring format](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt).

It should one line short description, than a long description.

File: blocks/bricks/simple.py [hunk 3]
```diff
@@ -284,6 +284,29 @@ def apply(self, input_):
...
+
+    .. math:: f(x) = \text{max}(x, ax)
+
+    Parameters
+    ----------
+    leak, optional : float
```
Comment (dmitriy-serdyuk) [inline blocks/bricks/simple.py]: See a link above, the syntax for parameters is different.

**Comment by dmitriy-serdyuk** [comment]
Ok, looks good to me.

**Pull request merged**